dts/xtensa/intel/intel_cavs25.dtsi

```diff
@@ -41,7 +41,7 @@
 	sram0: memory@be000000 {
 		device_type = "memory";
 		compatible = "mmio-sram";
-		reg = <0xbe000000 DT_SIZE_K(3008)>;
+		reg = <0xbe000000 DT_SIZE_K(1920)>;
```

Comment:
Do we need to have a comment in here making it obvious this is for TGL-H variant ?

Reply:
@lgirdwood we didn't have one before, saying, that it was the LP variant. We'll have to create additional boards and change naming anyway. We have more configurations than just one for each cAVS version. Would be good to record this somewhere - should I create an issue on github or do we prefer to track TODO items elsewhere?